microk8s-resources/wrappers/microk8s.wrapper

```diff
@@ -27,6 +27,9 @@ elif [ "${APP}" == "inspect" ]; then
 elif [ "${APP}" == "help" ] || [ "${APP}" == "--help" ] || [ "$APP" == "-h" ]; then
     help
     readonly EXIT="0"
+elif [ "${APP}" == "--version" ]; then
+    "${SNAP}/microk8s-version.wrapper"
+    readonly EXIT="$?"
```

Comment:
Is this to have both `microk8s version` and `microk8s --version` work?

Reply:
Yes. I added it when we weren't sure between `--version` and `version`. Will remove it